models/brandDao.js

```diff
@@ -0,0 +1,58 @@
+const { appDataSource } = require("./datasource"); // (../models/datasource)가 아닌 이유는 무엇인가요?
+
+//전체 브랜드 조회
+const getAllBrands = async () => {
+  return await appDataSource.query(`
+    SELECT * FROM brands`);
+};
+//브랜드 아이디로 브랜드 조회
+const getBrand = async (brandId) => {
+  return await appDataSource.query(
+    `
+    SELECT * FROM brands WHERE id = ?`,
+    [brandId]
+  );
+};
+
+//브랜드 이름으로 브랜드 조회
+const getBrandByName = async (brandName) => {
+  return await appDataSource.query(
+    `
+    SELECT * FROM brands WHERE name =?`,
+    [brandName]
+  );
+};
```

Comment:
```suggestion
//브랜드 이름으로 브랜드 조회
const getBrandByName = async (brandName) => {
  return await appDataSource.query(
    `
    SELECT * FROM brands WHERE name = ? LIMIT 1`,
    [brandName]
  );
};
```
있는지 여부를 조회할 때 LIMIT 1을 걸어주면, 브랜드를 하나 찾고나서, 곧바로 검색을 중단할 수 있어 db 관리 측면에서 보다 효율적입니다 :)